DependencyInjection/EzPlatformUIExtension.php

```diff
@@ -73,4 +73,17 @@ private function prependImageVariations(ContainerBuilder $container)
         $container->prependExtensionConfig('ezpublish', $config);
         $container->addResource(new FileResource($imageConfigFile));
     }
+
+    /**
+     * Returns the translation domains used by the extension.
+     * @return array An array of extensions
+     */
+    public function getTranslationDomains()
+    {
+        return [
+            'bar', 'confirm', 'contentedit', 'contenttypeselector', 'dashboardblocks', 'editorial',
```

Comment:
Consensus: document how these could be loaded from a YAML in the translatable extensions documentation.